docs/articles/documentation/reference/testcafe-api/runner/run.md

```diff
@@ -33,6 +33,7 @@ Parameter         | Type    | Description
 `pageRequestTimeout` | Number | Time (in milliseconds) to wait for HTML pages. If the page isn't received within the specified period, an error is thrown. | `25000`
 `browserInitTimeout` | Number | Time (in milliseconds) for browsers to connect to TestCafe and report that they are ready to test. If one or more browsers fail to connect within this period, TestCafe throws an error. | `120000` for [local browsers](../../command-line-interface.md#local-browsers), `360000` for [remote browsers](../../command-line-interface.md#remote-browsers)
 `speed`           | Number  | Specifies the test execution speed. A number between `1` (fastest) and `0.01` (slowest). If an individual action's speed is also specified, the action speed setting overrides the test speed. | `1`
+`cache` | Boolean | If enabled, the TestCafe proxy caches webpage assets (such as stylesheets, images and scripts) for the webpages that it processes. The next time the proxy accesses the page, it pulls assets from its cache instead of requesting them from the server. | `false`
 `stopOnFirstFail`    | Boolean | Defines whether to stop a test run if a test fails. You do not need to wait for all the tests to finish to focus on the first error. | `false`
```

Comment:
`to finish to focus` Explain. I suspect this is a similar case as my previous comment.

Reply:
A bit messy. The idea here is:

Usually, TestCafe runs all tests even if some of them fail along the way. With this option enabled, TestCafe runs tests until the first failure (so you could focus on problems one-by-one)

BTW can we safely omit `Defines whether to` here?

```suggestion
`stopOnFirstFail`    | Boolean | Stop the test run if a test fails. You can focus on the first error before all the tests finish. | `false`
```

OR


```suggestion
`stopOnFirstFail`    | Boolean | Stop the test run if a test fails. You do not need to wait until all tests in the suite finish before you can fix the first error. | `false`
```

Reply:
Stop**s** since this applies to just one. The rest is fine.